Tools/autotest/autotest.py

```diff
@@ -650,8 +650,8 @@ def list_subtests(*args, **kwargs):
 if __name__ == "__main__":
     ''' main program '''
     os.environ['PYTHONUNBUFFERED'] = '1'
-
-    os.putenv('TMPDIR', util.reltopdir('tmp'))
+    if sys.platform == "TMPDIR":
```

Comment:
if sys.platform != 'darwin' ?

Reply:
Or, you know, kill the delta since it's apparently not required? :-)

Reply:
yeap I will remove this line entirely